styx-flyte-client/src/test/java/com/spotify/styx/flyte/client/FlyteAdminClientTest.java

```diff
@@ -76,10 +76,39 @@ public void setUp() throws IOException {
   @Test
   public void shouldPropagateCreateExecutionToStub() {
     var workflowExecution =
-        flyteAdminClient.createExecution(DOMAIN, PROJECT, LP_IDENTIFIER,
+        flyteAdminClient.createExecution(PROJECT, DOMAIN, LP_IDENTIFIER,
             ExecutionOuterClass.ExecutionMetadata.ExecutionMode.SCHEDULED);
+    assertThat(PROJECT, equalTo(workflowExecution.getId().getProject()));
     assertThat(DOMAIN, equalTo(workflowExecution.getId().getDomain()));
+    assertThat(TestAdminService.WF_EXECUTION_ID_1, equalTo(workflowExecution.getId().getName()));
+  }
+
+  @Test
+  public void shouldPropagateGetExecutionToStub() {
+    var workflowExecution =
+        flyteAdminClient.getExecution(PROJECT, DOMAIN, LP_NAME);
     assertThat(PROJECT, equalTo(workflowExecution.getId().getProject()));
-    assertThat(TestAdminService.WF_EXECUTION_ID, equalTo(workflowExecution.getId().getName()));
+    assertThat(DOMAIN, equalTo(workflowExecution.getId().getDomain()));
+    assertThat(TestAdminService.WF_EXECUTION_ID_1, equalTo(workflowExecution.getId().getName()));
+  }
+
+  @Test
+  public void shouldPropagateTerminateExecutionToStub() {
+    var terminationResponse =
+        flyteAdminClient.terminateExecution(PROJECT, DOMAIN, LP_NAME, "Cause of termination");
+  }
```

Comment:
Assert at least that it is not a null value

```suggestion
    assertThat(terminationResponse, Matchers.notNullValue());
  }
```